functions/openhab.bash

```diff
@@ -63,11 +79,12 @@ Check the \"openHAB Release Notes\" and the official announcements to learn abou
   if ! cond_redirect apt-get ${APT_INST_OPTS} install "openhab2=${openhabVersion}"; then echo "FAILED (apt)"; exit 1; fi
   cond_redirect adduser openhab gpio
   cond_redirect systemctl daemon-reload
+  cp "${BASEDIR}"/includes/start-level.script "${OPENHAB_USERDATA}"/etc/scripts/
```

Comment:
should this be standard for all installs or part of the delayed rules option?

Reply:
I thought about that beforehand and think it should be part of any install hence I've put it here.